src/routes/write-exercise/components/InteractionSection.tsx

```diff
@@ -48,15 +47,14 @@ const InteractionSection = (props: InteractionSectionProps): ReactElement => {
   const isCorrectAlternativeSubmitted =
     isCorrect && stringSimilarity.compareTwoStrings(input, stringifyDocument(document)) <= ttsThreshold
   const submittedAlternative = isCorrectAlternativeSubmitted ? input : null
-
   const textInputRef = useRef<View>(null)
 
   useEffect(() => {
     if (!isAnswerSubmitted) {
       setInput('')
+      setSubmittedInput('')
     }
   }, [isAnswerSubmitted])
-
```

Comment:
This newline should stick, as it is between to new functions and makes the file better readable.